### PR TITLE
Add B(s) -> l l gamma

### PR DESCRIPTION
**Pull request by mreboud**
Implement the B_(s) -> \ell \ell \gamma decays.

### CODE REVIEW

File: flavio/data/parameters_metadata.yml [hunk 1]
```diff
@@ -674,6 +674,43 @@ Gamma12_B0_a:
...
+Bs->gamma KM betata:
+  tex: $\beta_{ta}$
+  description: beta parameter for Fta form factor in KM parametrization
+Bs->gamma KM deltata:
+  tex: $\Delta_{ta}$
+  description: Delta parameter for Fta form factor in KM parametrization
+
+
+# Parameters for the Bs->phi form factors (1503.05534v3)
+Bs->phi LCSR T10:
```
Comment (DavidMStraub) [inline flavio/data/parameters_metadata.yml]: This shouldn't be needed as `Bs->phi` form factors are already implemented in flavio. This parameter should actually be equal to the existing parameter `Bs->phi BSZ a0_T1` (which does not show up in this file as it is created programatically in `bsz.py`, but shows up on the [parameter list](https://flav-io.github.io/docs/parameters.html).

File: flavio/data/parameters_metadata.yml [hunk 2]
```diff
@@ -674,6 +674,43 @@ Gamma12_B0_a:
...
+  tex: $T_1(0)$
+  description: Bs->phi T_1 parameter at q2=0 
+Gamma_phi:
+  tex: $\Gamma_\phi$
```
Comment (DavidMStraub) [inline flavio/data/parameters_metadata.yml]: This shouldn't be necessary as there is already a parameter `tau_phi` (the lifetime) that is just the inverse width.

File: flavio/physics/bdecays/bllgamma.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        _process_tex =  r"B_s\to" +_tex[l]+r"^+"+_tex[l]+r"^-\gamma"
+        _process_taxonomy = r'Process :: $b$ hadron decays :: FCNC decays :: $B\to \ell^+\ell^-\gamma$ :: $'
+
+
+        # binned branching ratio
+        _obs_name = "<dBR/dq2>(Bs->"+l+l+"gamma)"
+        _obs = Observable(name=_obs_name, arguments=['q2min', 'q2max'])
+        _obs.set_description(r"Binned differential branching ratio of $" + _process_tex + r"$")
+        _obs.tex = r"$\langle \frac{d\text{BR}}{dq^2} \rangle(" + _process_tex + r")$"
+        _obs.add_taxonomy(_process_taxonomy)
```
Comment (DavidMStraub) [inline flavio/physics/bdecays/bllgamma.py]: Here probably you want to add the `_process_tex` and be careful with dollar signs (there is a stray one at the end of `_process_taxonomy`)

File: flavio/data/parameters_metadata.yml (hunk 1 above)
Comment (mreboud) [reply flavio/data/parameters_metadata.yml]: Exact, I will change this!

File: flavio/data/parameters_metadata.yml (hunk 2 above)
Comment (mreboud) [reply flavio/data/parameters_metadata.yml]: Here again, I will make the change. (I made these naive adds to have a working implementation)

File: flavio/physics/bdecays/bllgamma.py (hunk 3 above)
Comment (mreboud) [reply flavio/physics/bdecays/bllgamma.py]: Did you mean trading ```_process_taxonomy``` for ```_process_taxonomy + _process_tex + r"$"```?

Comment (DavidMStraub) [reply flavio/physics/bdecays/bllgamma.py]: Looks good, yes.

**Comment by mreboud** [comment]
One question: Charmonium resonances are not implemented, which means that the user should not use a ```[q2min, q2max]``` range that includes them. Should I add a warning if the range is ill-defined?

**Comment by DavidMStraub** [comment]
Yes, a warning is probably a good idea (I did something similar for the QCDF corrections to B->K*ll).

**Comment by mreboud** [comment]
Apart from a test file, the implementation is complete. I wanted to add BaBar measurement (http://inspirehep.net/record/753579) but what they actually measured is not clear. They don't specify the q^2 range but say that they vetoed the charmonium resonances...

**Comment by DavidMStraub** [comment]
@mreboud, just to be sure I'm not forgetting to merge this: do I understand correctly you still plan to add a unit test before I should merge it? Thanks!

**Comment by mreboud** [comment]
You are right, I have been busy with other things and kind of forget to finish this. I will do it asap.

**Comment by mreboud** [comment]
I added a basic test file and did a few cross checks. I think that the code is ready to be merged unless you have any comments.

**Comment by DavidMStraub** [comment]
Looks great, thanks!

**Pull request merged**